### PR TITLE
#181: Added a service container to the CMS.

### PR DESCRIPTION
**Pull request by RChaubey16**
* Added a service container to the CMS which can be used to add services to the CMS.
* Used the Symfony Dependency Injection component.
* Testing steps 
   1. Run command `composer update`.
   2. Go to the `/demo-service` route.
   3. The current date and time will be visible on a white screen.
   4. This live date is being displayed via the date service included in the CMS.

### CODE REVIEW

File: src/Services/ServicesClass/DateService.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+namespace Cms\Services\ServicesClass;
+
+class DateService
+{
+  public function currentDate(){
+    $date = date('d/m/Y h:i:s');
+    return $date;
+  }
+}
```
Comment (Libbna) [inline src/Services/ServicesClass/DateService.php]: no new line at EOF

File: src/Services/ServicesClass/DemoService.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  public function helloWorld()
+  {
+    return "Hello World!\n";
+  }
+
+  public function currentDate(){
+    $date = date('d/m/Y h:i:s');
+    return $date;
+  }
+}
```
Comment (Libbna) [inline src/Services/ServicesClass/DemoService.php]: No new line at EOF - add new line

**Review by vbadkar — APPROVED** [review]

**Review by Libbna — CHANGES_REQUESTED** [review]

**Review by vbadkar — APPROVED** [review]
image
Working on my side.

File: src/Services/ServicesClass/DemoService.php (hunk 2 above)
Comment (RChaubey16) [reply src/Services/ServicesClass/DemoService.php]: @Libbna Updated the PR with requested changes. Please review it

**Review by omkar-pd — APPROVED** [review]
Working
[image](https://user-images.githubusercontent.com/81369580/152524299-fcf1b858-a973-4572-b434-a9788ed9dc99.png)

**Review by Libbna — APPROVED** [review]
image

LGTM.

**Pull request merged**